packages/framework-provider-kubernetes-infrastructure/src/infrastructure/helm-manager.ts

```diff
@@ -20,7 +20,7 @@ export class HelmManager {
     const l = scopeLogger('exec', this.logger)
     const cmd = `${this.BASE_COMMAND} ${args}`
     l.debug('Executing command', cmd)
-    return exec(this.BASE_COMMAND, args)
+    return execFile(this.BASE_COMMAND, args.split(' '))
```

Comment:
I did a quick research and `execFile` is more secure than `exec`. Also removed the codeql warning